enaml/qt/qt_widget.py

```diff
@@ -344,7 +344,11 @@ def do_drag(self):
             qimg = get_cached_qimage(drag_data.image)
             qdrag.setPixmap(QPixmap.fromImage(qimg))
         else:
-            qdrag.setPixmap(widget.grab())
+            qt_major_version = __version_info__[0]
+            if qt_major_version < 5:
```

Comment:
This could be shorter as
```python
if __version_info__ < (5, ):
``` 
to help readability you could also alias __version_info__ at import to mention qt_version if you want.

Sorry for not suggesting this before. I will merge as soon as you update.